cmd/root.go

```diff
@@ -31,6 +31,7 @@ var rootCmd = &cobra.Command{
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 		cmd.SilenceUsage = true
 		if viper.GetBool("DEBUG") {
+			fmt.Println("CLI version: " + version)
```

Comment:
```suggestion
			fmt.Fprintln(os.Stderr, "CLI version: " + version)
```

Debug logs typically go to stderr because we don't want it to interfere with actual command output.

Reply:
ah wasn't intending to actually add this log to the PR. will remove

Reply:
removed and rebased